doc/doxygen/common/style_ini.css

```
@@ -43,9 +47,8 @@ div.ini_global > div.item {
 }
 
 .item_advanced {
-	/* we are kind of abusing an unrelated variable here */
-	/* TODO maybe redefine? */
-	color: var(--nav-menu-foreground-color)
+
+	color: var(--item-advanced-foreground-color);
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)